core/src/CopyChip/CopyChip.less

```diff
@@ -0,0 +1,21 @@
+@import "../typography.less";
+@import "../colors.less";
+
+
+
+.CopyChip {
+  position: relative;
+  &:hover {
+    cursor: copy;
+  }
+
+  button {
+    display: inline-block;
+    border-radius: 28px;
```

Comment:
Do not alter the styles of the button. This breaks the consistency and understanding of that element.

Reply:
I removed border-radius but need inline-block to allow the button to sit adjacent to element that needs to be copied